Package.swift

```diff
@@ -0,0 +1,38 @@
+// swift-tools-version:4.0
+import PackageDescription
+
+let package = Package(
```

Comment:
SPM by default will assign a predefined minimum deployment version for each supported platforms unless explicitly specified it with `platforms`, for example: 

```swift
platforms: [
   .iOS(.v5),
   .macOS(.v10_7)
],
```

From the SPM docs: "This predefined deployment version will be the oldest deployment target version supported by the installed SDK for a given platform." See [docs](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html) (SupportedPlatform section) for more information.

In this case it seems to be only iOS 13.

Reply:
Adding the supported platform versions will also require a bump in swift-tools-version number from 4.0 to 5.1:

`// swift-tools-version:5.1`

Reply:
Good call man.  Thanks for this.